lib/rubocop/cop/mixin/minitest_exploration_helpers.rb

```diff
@@ -23,7 +23,7 @@ module MinitestExplorationHelpers
       private
 
       def test_class?(class_node)
-        class_node.parent_class && class_node.identifier.source.end_with?('Test')
+        class_node&.class_type? && class_node&.parent_class && class_node.identifier.source.end_with?('Test')
```

Comment:
Can you revert this change? Because it is assumed that the class node is passed as indicated in the argument name.